<launch>
  <arg name="name" default="turtlebot3"/>
  <arg name="set_prefix" default="" />

  <param name="gt_txt_name" value="/home/shiqing/kitti2bag-master/kitti2bag/turtlebot3/gt/233_gt.txt"/>
  <param name="laserscan_txt_name" value="/home/shiqing/kitti2bag-master/kitti2bag/turtlebot3/laserscan/233_laserscan.txt"/>
  <param name="prefix" value="$(arg set_prefix)" />

  <node name="$(arg name)_listener" pkg="turtlebot3_listener" type="turtlebot3_listener" required="true" output="screen"/>
</launch>
